### PR TITLE
BaSP-TG-24: Admin Unit Test - getAll & getById

### PR DESCRIPTION
**Pull request by matiasraviola**
[getAll]: tested when you try to get all the list, filter by each field and when the route is wrong
[getById]: tested when you try to get a valid and/or invalid ID.

### CODE REVIEW

File: src/tests/admins.test.js [hunk 1]
```diff
@@ -1 +1,53 @@
...
+  lastName: 'Haysar',
+  email: 'thaysar0@miitbeian.gov.cn',
+  password: 'E6HCIX',
+};
+
+describe('GET /admin', () => {
+  test('should get all the admins', async () => {
+    const response = await request(app).get('/api/admin').send();
+    expect(response.body.data).toBeDefined();
+    expect(response.status).toBe(200);
```
Comment (PilarFernandezC) [inline src/tests/admins.test.js:22]: You could add a message test here (`response.body.message`). I think you could add it in other test cases too

Comment (matiasraviola) [reply src/tests/admins.test.js:22]: done

**Review by PilarFernandezC — APPROVED** [review]

**Review by paurinaldi — CHANGES_REQUESTED** [review]
Please check that getById function has the catch logic we talked about at the planning, as you correctly did with the other controller you are testing (getAllAdmins).

**Comment by matiasraviola** [comment]
changes are already done

**Review by paurinaldi — APPROVED** [review]

**Pull request merged**